api/DocumentOrShadowRoot.json

```diff
@@ -314,6 +314,7 @@
             },
             "chrome": {
               "version_added": "53"
+              ,"version_removed" : "65"
```

Comment:
The comma should be on the line above

Reply:
Also this shouldn't have a space before the colon.

Reply:
Using leading commas creates cleaner diffs. It makes clear that only one line has been edited, not two.

Reply:
Regardless it will cause the Travis build to fail